firejail/Cryptocat.profile

```diff
@@ -0,0 +1,20 @@
+# Firejail profile for 
```

Comment:
I was thinking about leaving the rest of this comment blank due to the lack of a name here, but I figured that'd be a bit too